build.gradle

```diff
@@ -7,38 +7,45 @@
  */
 
 plugins {
-    // Apply the java-library plugin to add support for Java Library
-    id 'java-library'
+	// Apply the java-library plugin to add support for Java Library
+	id 'java-library'
 }
 
 repositories {
-    // Use jcenter for resolving dependencies.
-    // You can declare any Maven/Ivy/file repository here.
-    jcenter()
+	// Use jcenter for resolving dependencies.
+	// You can declare any Maven/Ivy/file repository here.
+	jcenter()
+	mavenCentral()  // Required for Lombok dependency
 }
 
 dependencies {
-    // This dependency is used internally, and not exposed to consumers on their own compile classpath.
-    implementation 'com.google.guava:guava:29.0-jre'
-    implementation 'com.google.code.gson:gson:2.8.6'
-    // Use JUnit test framework
-    testImplementation 'junit:junit:4.12'
+	// This dependency is used internally, and not exposed to consumers on their own compile classpath.
+	implementation 'com.google.guava:guava:29.0-jre'
+	implementation 'com.google.code.gson:gson:2.8.6'
+	// Use JUnit test framework
+	testImplementation 'junit:junit:4.12'
+
+	// Lombok
+	compileOnly 'org.projectlombok:lombok:1.18.12'
```

Comment:
Reason using Lombok? Because I hope to reduce the dependency as possible, please let me know what is it for...

Reply:
This is a compile only dependency and it aims to reduce boilerplate code by providing automatic generation for common beans methods (getters, setters, `equals`, `hashcode`, `toString`, etc.) by using annotations. 
It is not *really* required but it helps reducing code size and overhead for developers.
Removing it is absolutely possible but would require implementing all these boilerplate methods by hand, which introduces more code to maintain.

Reply:
I would advise against using lombok. This lib caused a fair share of problems in projects i worked on. IDE Integrations breaking, refactoring flows breaking, wierd side effects, generated methods not showing up in the javadoc, etc.
Just generate the boilerplate with your IDE. it will save you a lot of headache.